CI failure is unrelated, seems to fail for all builds at the moment.

```
  1: Traceback (most recent call last):
  1:   File "/home/runner/work/rosbridge_suite/rosbridge_suite/ros_ws/src/phnk3mmp7y/rosbridge_suite/rosbridge_server/test/websocket/test_cbor_raw.py", line 13, in <module>
  1:     from twisted.internet import reactor
  1:   File "/usr/lib/python3/dist-packages/twisted/internet/reactor.py", line 38, in <module>
  1:     from twisted.internet import default
  1:   File "/usr/lib/python3/dist-packages/twisted/internet/default.py", line 56, in <module>
  1:     install = _getInstallFunction(platform)
  1:   File "/usr/lib/python3/dist-packages/twisted/internet/default.py", line 44, in _getInstallFunction
  1:     from twisted.internet.epollreactor import install
  1:   File "/usr/lib/python3/dist-packages/twisted/internet/epollreactor.py", line 24, in <module>
  1:     from twisted.internet import posixbase
  1:   File "/usr/lib/python3/dist-packages/twisted/internet/posixbase.py", line 18, in <module>
  1:     from twisted.internet import error, udp, tcp
  1:   File "/usr/lib/python3/dist-packages/twisted/internet/tcp.py", line 31, in <module>
  1:     from twisted.internet._newtls import (
  1:   File "/usr/lib/python3/dist-packages/twisted/internet/_newtls.py", line 21, in <module>
  1:     from twisted.protocols.tls import TLSMemoryBIOFactory, TLSMemoryBIOProtocol
  1:   File "/usr/lib/python3/dist-packages/twisted/protocols/tls.py", line 41, in <module>
  1:     from OpenSSL.SSL import Error, ZeroReturnError, WantReadError
  1:   File "/usr/lib/python3/dist-packages/OpenSSL/__init__.py", line 8, in <module>
  1:     from OpenSSL import crypto, SSL
  1:   File "/usr/lib/python3/dist-packages/OpenSSL/crypto.py", line 1553, in <module>
  1:     class X509StoreFlags(object):
  1:   File "/usr/lib/python3/dist-packages/OpenSSL/crypto.py", line 1573, in X509StoreFlags
  1:     CB_ISSUER_CHECK = _lib.X509_V_FLAG_CB_ISSUER_CHECK
  1: AttributeError: module 'lib' has no attribute 'X509_V_FLAG_CB_ISSUER_CHECK'
  ```

--- a/rosbridge_library/src/rosbridge_library/capabilities/subscribe.py
+++ b/rosbridge_library/src/rosbridge_library/capabilities/subscribe.py
@@ -266,19 +266,6 @@ class Subscribe(Capability):
         self.basic_type_check(msg, self.unsubscribe_msg_fields)
 
         topic = msg["topic"]
-        if Subscribe.topics_glob is not None and Subscribe.topics_glob:
-            self.protocol.log("debug", "Topic security glob enabled, checking topic: " + topic)
-            match = False
-            for glob in Subscribe.topics_glob:
-                if (fnmatch.fnmatch(topic, glob)):
-                    self.protocol.log("debug", "Found match with glob " + glob + ", continuing unsubscription...")
-                    match = True
-                    break
-            if not match:
-                self.protocol.log("warn", "No match found for topic, cancelling unsubscription from: " + topic)
-                return
-        else:
-            self.protocol.log("debug", "No topic security glob, not checking unsubscription.")
 
         if topic not in self._subscriptions:
             return
@@ -303,22 +290,6 @@ class Subscribe(Capability):
 
         """
         # TODO: fragmentation, proper ids
-        if Subscribe.topics_glob and Subscribe.topics_glob:
-            self.protocol.log("debug", "Topic security glob enabled, checking topic: " + topic)
-            match = False
-            for glob in Subscribe.topics_glob:
-                if (fnmatch.fnmatch(topic, glob)):
-                    self.protocol.log("debug", "Found match with glob " + glob + ", continuing topic publish...")
-                    match = True
-                    break
-            if not match:
-                self.protocol.log("warn", "No match found for topic, cancelling topic publish to: " + topic)
-                return
-        else:
-            self.protocol.log("debug", "No topic security glob, not checking topic publish.")
-
-        if PYTHON2:
-            topic = unicode(topic)  # noqa: F821
 
         outgoing_msg = {u"op": u"publish", u"topic": topic}
         if compression=="png":

